WeatherForecast/WeatherForecast/Model/CoreLocationManager.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  CoreLocationManager.swift
+//  WeatherForecast
+//
+//  Created by 김용재 on 2023/03/31.
+//
+
+import UIKit
+import CoreLocation
+
+final class CoreLocationManager: NSObject {
+    
+    weak var locationDelegate: LocationDelegate?
+    var locationManager: CLLocationManager?
```

Comment:
```suggestion
    private var locationManager: CLLocationManager?
```
private으로 감추는것이 좋을 것 같아요 그리고 항상 init 될때마다 생성되어야 하는 프로퍼티라면 non-optional 인게 더 좋을 것 같아요.

Reply:
locationManager의 타입을 non-optional 타입으로 바꾸고, private으로 감추는 대신에 현재 위치를 return하는 메소드인 fetchCurrentLocaiton()를 추가하였습니다.